Makefile

```diff
@@ -214,22 +241,22 @@ bundle-build: ## Build the bundle image.
 bundle-push: ## Push the bundle image.
 	$(MAKE) docker-push OPERATOR_IMAGE=$(BUNDLE_IMG)
 
-.PHONY: opm
-OPM = ./bin/opm
-opm: ## Download opm locally if necessary.
-ifeq (,$(wildcard $(OPM)))
-ifeq (,$(shell which opm 2>/dev/null))
-	@{ \
-	set -e ;\
-	mkdir -p $(dir $(OPM)) ;\
-	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
-	curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/v1.15.1/$${OS}-$${ARCH}-opm ;\
-	chmod +x $(OPM) ;\
-	}
-else
-OPM = $(shell which opm)
-endif
-endif
+.PHONY: fix-csv-replaces
+fix-csv-replaces: $(YQ)
+	$(eval REPLACES_VERSION=$(shell curl -sSL -H "Accept: application/vnd.github+json" \
+               -H "Authorization: Bearer ${GH_TOKEN}" \
+               -H "X-GitHub-Api-Version: 2022-11-28" \
+               https://api.github.com/repos/Kuadrant/authorino-operator/releases/latest | \
+               jq -r '.name'))
+
+	@echo $(REPLACES_VERSION)
+	V="authorino-operator.$(REPLACES_VERSION)" $(YQ) eval '.spec.replaces = strenv(V)' -i bundle/manifests/authorino-operator.clusterserviceversion.yaml
+
+.PHONY: prepare-release
+prepare-release: 
+	$(MAKE) manifests bundle VERSION=$(VERSION) AUTHORINO_VERSION=$(AUTHORINO_VERSION)
+	sed -i 's|\(defaultAuthorinoImage\s\+string = "quay.io/kuadrant/authorino:\).*"|\1v'$(AUTHORINO_VERSION)'"|' controllers/constants.go
```

Comment:
This is fine for a first iteration. For a subsequent improvement, I was thinking... The `controllers.defaultAuthorinoImage` constant could be set by using `-ldflags` when we `go build`. Of course, the value we set should come from somewhere. This could be file that we here create in the root of the project. `make build` (as well as `make run`) would pick up the file and set the flag. If the file is missing, it would default to `quay.io/kuadrant/authorino:latest`. WDYT?

Reply:
I definitely prefer this over `sed` actually. We would also have to check for the presence of this file in the Dockerfile where we build the binary too right (https://github.com/Kuadrant/authorino-operator/blob/main/Dockerfile#L20) to make sure that the correct version is set when the build and push images pipeline runs?